lib/member_page.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class String
+  def tidy
+    gsub(/[[:space:]]+/, ' ').strip
+  end
+end
+
+class MemberPage < Scraped::HTML
+  # TODO: Add contact info
+  field :source do
+    url
+  end
+
+  field :image do
+    noko.at_css('.dep_img img/@src').text.tidy
+  end
+
+  field :name do
+    name_and_id.split('-')[0..-2]
+               .map(&:upcase)
+               .join(' ')
+  end
+
+  field :name__ru do
+    noko.at_css('.dep_info h1').text.tidy
+  end
+
+  field :id do
+    name_and_id.split('-').last
+  end
+
+  field :area do
+    area_and_id[0]
+  end
+
+  field :area_id do
+    area_and_id[1]
+  end
+
+  field :term do
+    terms[russian_term_name.to_sym]
+  end
+
+  field :term_id do
+    term_ids[russian_term_name.to_sym]
+  end
+
```

Comment:
These seem a little over-complicated, when there's only a single entry in those hashes. (And from EP's POV we also only need the ID of the term, not its name…)